src/test/java/com/meltmedia/jgroups/aws/WithEc2.java

```diff
@@ -0,0 +1,26 @@
+package com.meltmedia.jgroups.aws;
+
+import com.amazonaws.services.ec2.AmazonEC2;
+import com.amazonaws.services.ec2.model.*;
+
+import java.util.Arrays;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public interface WithEc2 {
+  default AmazonEC2 ec2Mock(Tag... tags) {
```

Comment:
It just seems like you get the same thing from static test imports.  I can see how this might avoid large lists of static imports, when code formatters help out by expanding `*`.  You have taken your time to put your mark on this project, so I am not going to take a hard stance on this.  It is a style issue.  Just let me know when you think the code is ready to be merged.